draft-ietf-mls-protocol.md

```diff
@@ -1639,30 +1640,13 @@ the group.
 It is RECOMMENDED for the application generating exported values
 to refresh those values after a Commit is processed.
 
-## Recovery Keys
-
-The main MLS key schedule provides a `recovery_secret` which can provide extra
-security in some cross-group operations.
-
-The application SHOULD specify an upper limit on the number of past
-epochs for which the `recovery_secret` may be stored.
-
-There are three ways in which a `recovery_secret` can be used: to re-initialize
-the group with different parameters, to re-add lost group members or to create a
-sub-group of an existing group, as detailed in {{pre-shared-keys}}.
-
-Recovery keys are distinguished from exporter keys in that they have specific
-use inside the MLS protocol, whereas the use of exporter secrets may be
-decided by an external application. They are thus derived separately to avoid
-key material reuse.
-
 ## State Authentication Keys
 
 The main MLS key schedule provides a per-epoch `authentication_secret`.
 If one of the parties is being actively impersonated by an attacker, their
 `authentication_secret` will differ from that of the other group members.
 Thus, members of a group MAY use their `authentication_secrets` within
-an out-of-band authentication protocol to detect attacks or ensure that they
+an out-of-band authentication protocol to ensure that they
 share the same view of the group.
```

Comment:
I think detecting attacks is a valid goal of the authentication keys, we shouldn't remove it.

Reply:
It's redundant. The second clause of the sentence just says "detect attacks" again in a different way.